content/applications/marketing/surveys.rst

```diff
@@ -185,6 +212,53 @@ and the columns depict the various activity types.
    A new survey cannot be created in this view, as it is solely for the purpose of creating and
    viewing scheduled activities.
 
+Create surveys
+==============
+
+Learn about all the different options and configurations that can be utilized when creating a survey
+in Odoo.
+
+.. seealso::
+   :doc:`surveys/create`
+
+Scoring surveys
+===============
+
+Discover how to measure a survey participant's performance, or overall satisfaction, with Odoo's
+detailed (and fully customizable) survey scoring options.
+
+.. seealso::
+   :doc:`surveys/scoring`
+
+Create questions
+================
+
+With Odoo *Surveys*, there are many question types and options to choose from, providing the ability
+to create any kind of unique survey, questionnarire, and/or certification.
+
+.. seealso::
+   :doc:`surveys/questions`
+
+Live Session surveys
+====================
+
+The *Live Session* survey option available in Odoo can enhance in-person demonstrations and
+presentations, where participants' real-time responses can be used to dictate where the conversation
+goes next.
+
+.. seealso::
+   :doc:`surveys/live_session`
+
+Survey analysis
+===============
+
+Once the surveys start to come in, it is time to analyze the responses from your participants.
+Fortuantely, the in-depth reporting pages and options available in Odoo *Surveys* provide countless
+ways to examine everything related to surveys, and their submitted responses.
+
+.. seealso::
+   :doc:`surveys/analysis`
+
```

Comment:
I am wondering if this section at the bottom might be a little confusing, since you already have the information (just abbreviated) in the cards at the top. You could either eliminate this section since the cards are doing the same work (an explanation with a link to the doc), and/or you can expand the text in the cards to include all of the text found in these sections.

Reply:
I fully understand (and appreciate) this comment. However, in a previous doc that took care of the same thing (adding cards/intro info to the main doc, etc.), Sam suggested that I add these sections at the bottom, with their own seealso admonitions. I'm assuming it's just in case readers bypassed the cards at the top, proceeded to scroll through, then hit the bottom, wondering where to go next. So, since Sam has suggested this in the past, I think I'm going to keep it in. But, that said, we'll see if he's changed his mind about it when it comes time for this PR to get a Tech Review 😄 Thanks again for the always in-depth feedback! 🙏